web3.js/src/errors.ts

```diff
@@ -0,0 +1,9 @@
+export class SendTransactionError extends Error {
+  logs: string | undefined;
+
+  constructor(message: string, logs?: string) {
```

Comment:
logs will either be undefined or an array of strings. Can you please update the type of `logs` everywhere to indicate that it's an array?

Reply:
oh yeah, of course